Simplified/UILabel+NYPLAppearanceAdditions.m

```diff
@@ -20,6 +20,7 @@ - (void)setFontName:(NSString *)fontName
   UIFont *currentFont = self.font;
   CGFloat fontSize = currentFont.pointSize;
   UIFont *newFont = [UIFont fontWithName:fontName size:fontSize];
+    // Shouldn't we return FALSE if no font with given name exists?
```

Comment:
I think a setter should return `void`, otherwise this can no longer be a property implementation. The caller can always check using the property getter.

Reply:
Agreed